Docker image for CPU

## Introduction
We don't really need GPU for `convert` part, but the current docker image provided by us needs GPU driver to run.
I think it is better to provide a docker image for CPU.

## What is preventing this?
If we run `convert` with tensorflow-cpu, following error occurred.
```
self = <tensorflow.python.client.session.Session object at 0x7f26eaf44d30>
fn = <function BaseSession._do_run.<locals>._run_fn at 0x7f26f408abf8>
args = ({<tensorflow.python.pywrap_tensorflow_internal.TF_Output; proxy of <Swig Object of type 'TF_Output *' at 0x7f26ef166c...ywrap_tensorflow_internal.TF_Output; proxy of <Swig Object of type 'TF_Output *' at 0x7f26ef00be10> >], [], None, None)
message = "Node 'conv1/BatchNorm/FusedBatchNormV3' (type: '_FusedConv2D', num of outputs: 1) does not have output 1"
m = None

    def _do_call(self, fn, *args):
      try:
        return fn(*args)
      except errors.OpError as e:
        message = compat.as_text(e.message)
        m = BaseSession._NODEDEF_NAME_RE.search(message)
        node_def = None
        op = None
        if m is not None:
          node_name = m.group(3)
          try:
            op = self._graph.get_operation_by_name(node_name)
            node_def = op.node_def
          except KeyError:
            pass
        message = error_interpolation.interpolate(message, self._graph)
        if 'only supports NHWC tensor format' in message:
          message += ('\nA possible workaround: Try disabling Grappler optimizer'
                      '\nby modifying the config for creating the session eg.'
                      '\nsession_config.graph_options.rewrite_options.'
                      'disable_meta_optimizer = True')
>       raise type(e)(node_def, op, message)
E       tensorflow.python.framework.errors_impl.OutOfRangeError: Node 'conv1/BatchNorm/FusedBatchNormV3' (type: '_FusedConv2D', num of outputs: 1) does not have output 1

/usr/local/pyenv/versions/3.6.3/envs/python3.6/lib/python3.6/site-packages/tensorflow_core/python/client/session.py:1384: OutOfRangeError
```
The details of this error can see #106 .
This error happened in `lmnet/export` and this feature is for debugging and testing. 
https://github.com/blue-oil/blueoil/blob/4ad07e2ea85b005188f1fa0db1cc1d0daefcc1d1/lmnet/executor/export.py#L112-L137

According to comments, this is for the test or debug. So, we can avoid this error and provide CPU version, I think.
To avoid this error, we already treat this as using flag(save_npy_for_debug)
https://github.com/blue-oil/blueoil/blob/4ad07e2ea85b005188f1fa0db1cc1d0daefcc1d1/blueoil/cmd/convert.py#L160-L164

So it's easy to change the default value of `save_npy_for_debug` to `False`.
https://github.com/blue-oil/blueoil/blob/4ad07e2ea85b005188f1fa0db1cc1d0daefcc1d1/blueoil/cmd/convert.py#L209

## Tasks
If the above change is approved, I will do tasks as following

- [ ] Change the default value of `save_npy_for_debug` to `False` and enable to change this via CLI option
- [ ] Create Dockerfile for CPU and replace docker image for tests (run all tests on this image)
- [ ] Add test for Dockerfile for GPU (run e2e test once)

--- a/docs/tutorial/image_cls.md
+++ b/docs/tutorial/image_cls.md
@@ -140,7 +140,7 @@ Convert trained model to executable binary files for x86, ARM, and FPGA.
 Currently, conversion for FPGA only supports Intel Cyclone® V SoC FPGA.
 
     $ docker run --rm \
-        -e CUDA_VISIBLE_DEVICES=-1\
+        -e CUDA_VISIBLE_DEVICES=-1 \
         -e OUTPUT_DIR=/home/blueoil/saved \
         -v $(pwd)/saved:/home/blueoil/saved \
         blueoil_$(id -un):{TAG} \

--- a/docs/tutorial/image_cls.md
+++ b/docs/tutorial/image_cls.md
@@ -140,7 +140,7 @@ Convert trained model to executable binary files for x86, ARM, and FPGA.
 Currently, conversion for FPGA only supports Intel Cyclone® V SoC FPGA.
 
     $ docker run --rm \
-        -e CUDA_VISIBLE_DEVICES=0 \
+        -e CUDA_VISIBLE_DEVICES=-1\
         -e OUTPUT_DIR=/home/blueoil/saved \
         -v $(pwd)/saved:/home/blueoil/saved \
         blueoil_$(id -un):{TAG} \
@@ -152,6 +152,7 @@ Currently, conversion for FPGA only supports Intel Cyclone® V SoC FPGA.
 - Optimizes graph.
 - Generates source code for executable binary.
 - Compiles for x86, ARM and FPGA.
+- CUDA_VISIBLE_DEVICES=-1 means converting with CPU
 
 If conversion is successful, output files are generated under `./saved/{MODEL_NAME}/export/save.ckpt-{Checkpoint No.}/{Image size}/output`.
 

--- a/docs/tutorial/image_det.md
+++ b/docs/tutorial/image_det.md
@@ -104,7 +104,7 @@ Convert trained model to executable binary files for x86, ARM, and FPGA.
 Currently, conversion for FPGA only supports Intel Cyclone® V SoC FPGA.
 
     $ docker run --rm \
-	    -e CUDA_VISIBLE_DEVICES=0 \
+	    -e CUDA_VISIBLE_DEVICES=-1 \
 	    -e OUTPUT_DIR=/home/blueoil/saved \
 	    -v $(pwd)/saved:/home/blueoil/saved \
 	    blueoil_$(id -un):{TAG} \
@@ -115,6 +115,7 @@ Currently, conversion for FPGA only supports Intel Cyclone® V SoC FPGA.
 - Optimizes graph.
 - Generates source code for executable binary.
 - Compiles for x86, ARM and FPGA.
+- CUDA_VISIBLE_DEVICES=-1 means converting with CPU
 
 If conversion is successful, output files are generated under `./saved/{MODEL_NAME}/export/save.ckpt-{Checkpoint No.}/{Image size}/output`.
 

--- a/docs/tutorial/image_keypoint_detection.md
+++ b/docs/tutorial/image_keypoint_detection.md
@@ -103,7 +103,7 @@ Convert trained model to executable binary files for x86, ARM, and FPGA.
 Currently, conversion for FPGA only supports Intel Cyclone® V SoC FPGA.
 
     $ docker run --rm \
-        -e CUDA_VISIBLE_DEVICES=0 \
+        -e CUDA_VISIBLE_DEVICES=-1 \
         -e OUTPUT_DIR=/home/blueoil/saved \
         -v $(pwd)/saved:/home/blueoil/saved \
         blueoil_$(id -un):{TAG} \
@@ -114,6 +114,7 @@ Currently, conversion for FPGA only supports Intel Cyclone® V SoC FPGA.
 - Optimizes graph.
 - Generates source code for executable binary.
 - Compiles for x86, ARM and FPGA.
+- CUDA_VISIBLE_DEVICES=-1 means converting with CPU
 
 If conversion is successful, output files are generated under
 `./saved/{MODEL_NAME}/export/save.ckpt-{Checkpoint No.}/{Image size}/output`.

--- a/docs/tutorial/image_seg.md
+++ b/docs/tutorial/image_seg.md
@@ -133,7 +133,7 @@ Convert trained model to executable binary files for x86, ARM, and FPGA.
 Currently, conversion for FPGA only supports Intel Cyclone® V SoC FPGA.
 
     $ docker run --rm \
-	    -e CUDA_VISIBLE_DEVICES=0 \
+	    -e CUDA_VISIBLE_DEVICES=-1 \
 	    -e OUTPUT_DIR=/home/blueoil/saved \
 	    -v $(pwd)/saved:/home/blueoil/saved \
 	    blueoil_$(id -un):{TAG} \
@@ -144,6 +144,7 @@ Currently, conversion for FPGA only supports Intel Cyclone® V SoC FPGA.
 - Optimize graph
 - Generate source code for executable binary
 - Compile for x86, ARM and FPGA
+- CUDA_VISIBLE_DEVICES=-1 means converting with CPU
 
 If conversion is successful, output files are generated under `./saved/{MODEL_NAME}/export/save.ckpt-{Checkpoint No.}/{Image size}/output`.
 

